lib/messages/subscription-list.js

```diff
@@ -18,28 +19,40 @@ module.exports = class SubscriptionList extends Message {
       attachments: [{
         ...this.getBaseMessage(),
         fallback: `${prefix} ${this.repositories.length} repositor${this.repositories.length === 1 ? 'y' : 'ies'}`,
+        title: `${prefix} these repositories`,
+        text: this.repositories.length > 0 ? this.repositoriesToString().join('\n') : 'no repositories',
+      }, {
+        ...this.getBaseMessage(),
+        fallback: `${prefix} ${this.accounts.length} account${this.accounts.length === 1 ? '' : 's'}`,
+        title: `${prefix} these accounts`,
+        text: this.accounts.length > 0 ? this.accountsToString().join('\n') : 'no accounts',
```

Comment:
Looks like this currently works out to the following in case of no repos and no accounts: 
![image](https://user-images.githubusercontent.com/7718702/46754313-0f735480-ccba-11e8-8d8f-14d72cc46ad2.png)

Could we change this so that if a channel is not subscribed to any repositories, only the list for accounts is shown? And similarly if a channel is not subscribed to any accounts, only the list of repositories is shown?

If a channel is not subscribed to any repositories and also not subscribed to any accounts can we say something like: "#channel is not subscribed to any repositories or accounts"?